crates/db_schema/src/source/post.rs

```diff
@@ -192,11 +194,14 @@ pub struct PostSaved {
   pub published: DateTime<Utc>,
 }
 
+#[derive(derive_new::new)]
 #[cfg_attr(feature = "full", derive(Insertable, AsChangeset))]
 #[cfg_attr(feature = "full", diesel(table_name = post_actions))]
 pub struct PostSavedForm {
   pub post_id: PostId,
   pub person_id: PersonId,
+  #[new(value = "Utc::now()")]
+  pub saved: DateTime<Utc>,
```

Comment:
Main change is in this file:

- Adding `derive_new` to all of these forms.
- Adding a default `Utc::now()` to the timestamp column.